source/server-selection/server-selection.rst

```diff
@@ -521,6 +527,24 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
+`````
+
+The read preference ``hedge`` parameter is a document that configures how the
+server will perform hedged reads. It consists of the following keys:
+
+- ``enabled`` (default true): This enables hedging
+- ``delay`` (default true): This enables staggered reads
+
+To explicitly enable hedging, an empty document must be sent to the server for
+the `hedge` option. This enables hedging with the default values specified
+above. To not enable hedging, the `hedge` option must be omitted entirely.
+
+Drivers MUST NOT fill in default values for any keys in the ``hedge`` document
+if they are omitted. Drivers MUST raise an error if ``hedge`` is set, and the
+server's ``maxWireVersion`` is less than 9.
```

Comment:
As written, I'm not sure how this would play out when resolving a read preference as part of server selection. If the client is connected to a non-sharded topology, I suppose `hedge` would simply never get used and could be ignored. Unless you think server selection should raise a client-side error if it sees `hedge` and the topology is non-sharded -- in which case we could revisit that when/if drivers support `hedge` on their own.

Since we only send `readPreference` to mongos nodes, perhaps we should rephase this as:

> Drivers MUST NOT send a read preference where the `hedge` option is set to server version before 4.4 (i.e. `maxWireVersion` < 9) and MUST raise an error instead. This pertains to [passing a read preference to mongos](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#passing-read-preference-to-mongos).

Reply:
I don't think drivers should go as far as checking for the topology in order to raise an error for an unsupported `hedge` option. Checking the wire version should suffice.

Reply:
I concur. Since the new text refers to sending for mongos, I'm good to resolve this.